buildpacks/typescript/bin/build

```diff
@@ -13,8 +13,12 @@ layers_dir=$1
 # shellcheck source=/dev/null
 source "$bp_dir/lib/build.sh"
 
+store_node_version "$layers_dir"
+
 clear_cache_on_stack_change "$layers_dir"
 
+clear_cache_on_node_version_change "$layers_dir"
```

Comment:
This is actually unnecessary in the typescript buildpack since node isnt installed, right? Can the changes in the ts buildpack?